composer.json

```diff
@@ -31,7 +31,8 @@
         "doctrine/dbal": "~2.3",
         "jdorn/sql-formatter": "~1.1",
         "symfony/doctrine-bridge": "~2.2|~3.0",
-        "doctrine/doctrine-cache-bundle": "~1.0"
+        "doctrine/doctrine-cache-bundle": "~1.0",
+        "symfony/web-profiler-bundle": "~2.7|~3.0"
```

Comment:
> Dropping Symfony 2.3 would probably implicitly ensure it is

What's blocking this then? :-)